tools/blue_deployment/step1.js

```diff
@@ -0,0 +1,343 @@
+ * @param {string} remoteStateRoot The state root of the remote chain to anchor.
+ * @param {number} maxStateroots The max number of state roots to store in
+ *                               thecircular buffer.
+ *
+ * @returns {Contract} The Anchor contract instance.
+ */
+const createAnchor = (
+    organization,
+    remoteChainId,
+    remoteBlockHeight,
+    remoteStateRoot,
+    maxStateroots = '10',
+) => {
+    return Contract.loadTruffleContract(
+        'Anchor',
+        [
+            remoteChainId,
+            remoteBlockHeight,
+            remoteStateRoot,
+            maxStateroots,
+            organization,
+        ],
+        { rootDir },
+    );
+};
+
+const createEIP20Gateway = (
+    tokenAddress,
+    baseTokenAddress,
+    anchorAddress,
+    bounty,
+    organizationAddress,
+    burnerAddress = '0x0000000000000000000000000000000000000000',
+) => {
+    return Contract.loadTruffleContract(
+        'EIP20Gateway',
+        [
+            tokenAddress,
+            baseTokenAddress,
+            anchorAddress,
+            bounty,
+            organizationAddress,
+            burnerAddress,
+        ],
+        { rootDir },
+    );
+};
+
+// tokenAddress = Branded Token
+// baseTokenAddress = OST
+const deployAnchorAndGateway = async (
+    rpcEndpointOrigin,
+    deployerAddress,
+    tokenAddress,
+    baseTokenAddress,
+    bounty,
+    chainIdAuxiliary,
+    blockHeightAuxiliary,
+    stateRootAuxiliary,
+) => {
+    const web3 = new Web3(rpcEndpointOrigin);
+    const startingNonce = await web3.eth.getTransactionCount(deployerAddress);
+
+    const {
+        MerklePatriciaProof,
+        GatewayLib,
+        MessageBus,
+    } = createLibraryConracts();
+    const Organization = createOrganization(
+        deployerAddress, // TODO
+        deployerAddress, // TODO
+        [deployerAddress], // TODO
+        '100000000000', // TODO
+    );
+    const Anchor = createAnchor(
+        Organization.reference(),
+        chainIdAuxiliary,
+        blockHeightAuxiliary,
+        stateRootAuxiliary,
+        '10', // TODO
+    );
+
+    const EIP20Gateway = createEIP20Gateway(
+        tokenAddress,
+        baseTokenAddress,
+        Anchor.reference(),
+        bounty,
+        Organization.reference(),
+        '0x0000000000000000000000000000000000000000', // TODO: burner address
+    );
+    EIP20Gateway.addLinkedDependency(GatewayLib);
+    EIP20Gateway.addLinkedDependency(MessageBus);
+
+    const registry = new ContractRegistry();
+    registry.addContracts([
+        EIP20Gateway,
+        GatewayLib,
+        MerklePatriciaProof,
+        MessageBus,
+        Organization,
+        Anchor,
+    ]);
+
+    const deploymentObjects = registry.toLiveTransactionObjects(deployerAddress, startingNonce);
+    return deployContracts(
+        new UnlockedWeb3Signer(web3),
+        web3,
+        deploymentObjects,
+    );
+};
+
+
+// deploy for Auxiliary
+const deployAnchorAndCoGateway = async (
```

Comment:
Instead of the comment, you could rename the function to `deployAuxiliary` instead of `deployAnchorAndCoGateway`.

Reply:
👍